### PR TITLE
PORTALS-3027

### PR DESCRIPTION
**Pull request by nickgros**

### CODE REVIEW

File: apps/portals/src/configurations/elportal/synapseConfigs/computational_tools.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const computationalCardConfiguration: CardConfiguration = {
     {
       isMarkdown: false,
       baseURL: 'Explore/Projects/DetailsPage',
-      URLColumnName: 'grantNumber',
+      URLColumnName: 'grant',
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/computational_tools.ts:25]: Fix link to go from grant number on computational tools card to project page (where projects are uniquely identified by a set of grants)

image

File: apps/portals/src/configurations/elportal/synapseConfigs/projects.ts [hunk 2]
```diff
@@ -25,14 +25,14 @@ export const projectCardConfiguration: CardConfiguration = {
     title: 'name',
     subTitle: 'principalInvestigators',
     description: 'abstract',
-    secondaryLabels: ['institutions', 'program', 'grantNumber'],
+    secondaryLabels: ['institutions', 'program', 'grant'],
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/projects.ts:28]: Show the 'grant' on a project card: 
image

File: apps/portals/src/configurations/elportal/synapseConfigs/projects.ts [hunk 3]
```diff
@@ -25,14 +25,14 @@ export const projectCardConfiguration: CardConfiguration = {
...
   },
   secondaryLabelLimit: 4,
   titleLinkConfig: {
     isMarkdown: false,
     baseURL: 'Explore/Projects/DetailsPage',
-    URLColumnName: 'grantNumber',
-    matchColumnName: 'grantNumber',
+    URLColumnName: 'grant',
+    matchColumnName: 'grant',
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/projects.ts:35]: Card title on explore page should link to the details page using the `grant` value(s) as the unique identifier for the project.

File: apps/portals/src/configurations/elportal/synapseConfigs/projects.ts [hunk 4]
```diff
@@ -49,20 +49,20 @@ export const projectsDetailsPageConfiguration: DetailsPageProps = {
   synapseConfigArray: [
     {
       name: 'CardContainerLogic',
-      columnName: 'grantNumber',
+      columnName: 'grant',
       title: 'Studies',
-      tableSqlKeys: ['grantNumber'],
+      tableSqlKeys: ['grant'],
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/projects.ts:54]: The `studies` cards shown on the details page should be filtered using the `grant` column

File: apps/portals/src/configurations/elportal/synapseConfigs/projects.ts [hunk 5]
```diff
@@ -49,20 +49,20 @@ export const projectsDetailsPageConfiguration: DetailsPageProps = {
...
       },
     },
     {
       name: 'CardContainerLogic',
-      columnName: 'grantNumber',
+      columnName: 'grant',
       title: 'Publications',
       showTitleSeperator: false,
-      tableSqlKeys: ['Grant'],
+      tableSqlKeys: ['grant'],
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/projects.ts:65]: ditto for publications

File: apps/portals/src/configurations/elportal/synapseConfigs/projects.ts [hunk 6]
```diff
@@ -82,7 +82,7 @@ export const projectsDetailsPageConfiguration: DetailsPageProps = {
     {
       name: 'CardContainerLogic',
       title: 'Computational Tools',
-      columnName: 'grantNumber',
+      columnName: 'grant',
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/projects.ts:85]: ditto for computational tools

File: apps/portals/src/configurations/elportal/synapseConfigs/projects.ts [hunk 7]
```diff
@@ -106,7 +106,7 @@ const projects: SynapseConfig = {
       'name',
       'institutions',
       'principalInvestigators',
-      'grantNumber',
+      'grant',
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/projects.ts:109]: Plot `'grant'` on the projects explore page: 
image

File: apps/portals/src/configurations/elportal/synapseConfigs/studies.ts [hunk 8]
```diff
@@ -28,8 +28,8 @@ export const studyCardConfiguration: CardConfiguration = {
     {
       isMarkdown: false,
       baseURL: 'Explore/Projects/DetailsPage',
-      URLColumnName: 'grantNumber',
-      matchColumnName: 'grantNumber',
+      URLColumnName: 'grant',
+      matchColumnName: 'grant',
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/studies.ts:32]: Studies linking to the project details page should use the `grant` column to determine the project's unique identifier

File: apps/portals/src/configurations/elportal/synapseConfigs/studies.ts [hunk 9]
```diff
@@ -54,7 +54,6 @@ export const studyCardConfiguration: CardConfiguration = {
 const columnAliases = {
   dataTypeAll: 'Data Types',
   Number_of_Individuals: 'Individuals',
-  'Grant Number': 'Grant',
```
Comment (nickgros) [inline apps/portals/src/configurations/elportal/synapseConfigs/studies.ts:57]: Remove unnecessary column alias

**Review by hallieswan — APPROVED** [review]

**Pull request merged**